.github/workflows/ci.yml

```diff
@@ -0,0 +1,43 @@
+name: Python CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
Ned has suggested that we run tests on all PRs, not just the ones targeting master: https://edx-internal.slack.com/archives/CD89W7Z7F/p1606849364465500